### PR TITLE
First attempt at group quickstart instructions

### PR DESCRIPTION
**Pull request by OisinKyne**

### CODE REVIEW

File: docs/int/quickstart/quickstart-alone.md [hunk 1]
```diff
@@ -65,20 +58,6 @@ There are two ways to test out a distributed validator.
 Congratulations, if this all worked you are now running a distributed validator cluster on a testnet. Try turning off a single node of the four and see if the validator stays online or begins missing duties, to see for yourself the fault-tolerance that can be added to proof of stake validation with this new Distributed Validator Technology. 
 
 :::tip 
-Don't forget to be a good testnet steward and exit your validator when you are finished testing with it.* 
-
-**Once charon creates validator exit data in an upcoming release.*
+Don't forget to be a good testnet steward and exit your validator when you are finished testing with it.
```
Comment (dB2510) [inline docs/int/quickstart/quickstart-alone.md:61]: suggest: also include steps to exit validator or maybe a link for reference

File: docs/int/quickstart/quickstart-group.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+:::caution
+Charon is in an early alpha state and is not ready to be run on mainnet
+:::
+
+To create a distributed validator cluster with a group of other node operators requires four key steps:
+- Every operator prepares their software and gets their charon client's ENR
+- One operator prepares the terms of the distributed validator key generation ceremony
+    - They select the network, the withdrawal address, the number of 32 ether distributed validators to create, and the ENRs of each operator taking part in the ceremony.
+    - In future, the DV launchpad will facilitate this process more seamlessly, with consent on the terms provided by all operators that participate.
+- Every operator runs the ceremony, and once successful, a number of cluster artifacts are created, including:
```
Comment (dB2510) [inline docs/int/quickstart/quickstart-group.md:17]:
```suggestion
- Every operator participates in the DKG ceremony, and once successful, a number of cluster artifacts are created, including:
```

File: docs/int/quickstart/quickstart-group.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+    # Create .charon/cluster-definition.json to participate in DKG ceremony
+    docker run --rm -v "$(pwd):/opt/charon" --env-file .env ghcr.io/obolnetwork/charon:latest create dkg
+    ```
+1. The operator that ran this command should distribute the resulting `cluster-definition.json` file to each operator.
+
+1. At a pre-agreed time, all operators run the ceremony program with the `charon dkg` command
+    ```sh
+    # Copy the cluster-definition.json file to .charon
+    cp cluster-definition.json .charon/
```
Comment (dB2510) [inline docs/int/quickstart/quickstart-group.md:83]: i think we don't need this, if everyone runs the commands as mentioned then `create dkg` command by default creates `cluster-definition.json` inside .charon

File: docs/int/quickstart/quickstart-group.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    - They select the network, the withdrawal address, the number of 32 ether distributed validators to create, and the ENRs of each operator taking part in the ceremony.
+    - In future, the DV launchpad will facilitate this process more seamlessly, with consent on the terms provided by all operators that participate.
+- Every operator runs the ceremony, and once successful, a number of cluster artifacts are created, including:
+    - The private key shares for each distributed validator
+    - The deposit data file containing deposit details for each distributed validator
+    - A `cluster-lock.json` file which contains the finalised terms of this cluster required by charon to operate. 
+- Every operator starts their node with `charon run`, and uses their monitoring to determine the cluster health and connectivity
+- Once the cluster is confirmed to be healthy, deposit data files created during this process are activated on the staking launchpad. 
+
+### Getting setup with charon
```
Comment (dB2510) [inline docs/int/quickstart/quickstart-group.md:24]:
```suggestion
# Getting started with a single distributed validator charon node
```
Something sort of like this to have a separate heading for running a single DV node

**Review by dB2510 — APPROVED** [review]
LGTM

File: docs/int/quickstart/quickstart-group.md (hunk 3 above)
Comment (OisinKyne) [reply docs/int/quickstart/quickstart-group.md:83]: If everyone ran `charon create dkg`, wouldn't they all have different UUIDs in the files and shouldn't the charon nodes fail the handshake? It has to be one cluster-definition.json shared to everyone, right?

**Pull request merged**